lib/logstash/inputs/stomp.rb

```diff
@@ -47,9 +53,21 @@ def connect
   public
   def register
     require "onstomp"
-    @client = OnStomp::Client.new("stomp://#{@host}:#{@port}", :login => @user, :passcode => @password.value)
+    # Set protocol based on wether ssl is true or not
+    if @ssl
+      @protocol = "stomp+ssl"
+      if @verify_ssl
+        @ssl_opts = {}  # default in onstomp is to verify certificates
+      else
+        @ssl_opts = {:verify_mode => OpenSSL::SSL::VERIFY_NONE, :post_connection_check => false}  # disable verification
+      end
+      @client = OnStomp::Client.new("stomp+ssl://#{@host}:#{@port}", :login => @user, :passcode => @password.value, :ssl => @ssl_opts)
```

Comment:
I'm having no luck finding the docs for the `:ssl` setting for OnStomp. Specifically, where does it look for trusted certificate authorities?

If there's no way to configure what certificates to trust, got any ideas on documenting what the default cert trust is?